rules/aws_cloudtrail_rules/aws_cloudtrail_account_discovery.py

```diff
@@ -0,0 +1,21 @@
+from panther_base_helpers import deep_get
+
+DISCOVERY_EVENTS = [
+    "GetAlternateContact",
+    "GetContactInformation",
+    "PutAlternateContact",
+    "PutContactInformation",
+    "DescribeAccount",
+]
+
+
+def rule(event):
+    return event.get("eventName") in DISCOVERY_EVENTS
+
+
+def title(event):
+    return (
+        f"User [{deep_get(event, 'userIdentity', 'principalId')}]"
```

Comment:
I would advise `userIdentity:arn` in place of `userIdentity:principalId`. `userIdentity:arn` conveys the account id and type of the entity making the request.

Reply:
fixed :)